### ISSUE TITLE
Creating a user defined GOSlim

### ISSUE THREAD

**Issue by woundhealing** [geneontology/helpdesk#34]
Hello,
I have some proteomics data in the context of wound healing and inflammation. To categorise these proteins, I would like to create a user-defined GOslim for this specific context.

I am quite new to GO and am unsure of what the current tools available for creating GOslims are. Is there a method for creating a GOslim without too much user bias? For example, taking the most enriched categories in the dataset and using those as GOslim terms?

My goal is to create a histogram with the number of proteins up- and down-regulated in each GOslim category. I just need to create those categories first.

Advice would be much appreciated!

**Comment by kltm** [comment]
@cmungall Do you think somebody in the ontology group could help out?

**Comment by mcourtot** [comment]
Hi @woundhealing,

We did something I believe similar to build a specific GO slim based on a metagenomics dataset, and visualise categories. The code and corresponding paper is available at https://github.com/EBISPOT/slim-o-matic

The idea was using the dataset available and generate the slim based on this. Note that this is a semi-automated method: you would still need to look at your generated slim manually and inspect whether it makes sense or not, and decide how to further trim. Maybe it can be useful as starting point?

We had discussed with @simonjupp the possibility of having this tool be included in OLS, https://github.com/EBISPOT/OLS - not sure whether this is in the pipeline or not.
